akka-docs/src/main/paradox/general/configuration.md

```diff
@@ -320,24 +320,24 @@ Java
 : @@snip [ConfigDocTest.java](/akka-docs/src/test/java/jdocs/config/ConfigDocTest.java) { #custom-config-2 }
 
 
-When working with `Config` objects, keep in mind that there are
+When working with [Config](https://lightbend.github.io/config/latest/api/com/typesafe/config/Config.html) objects, keep in mind that there are
 three "layers" in the cake:
 
- * `ConfigFactory.defaultOverrides()` (system properties)
+ * [ConfigFactory.defaultOverrides()](https://lightbend.github.io/config/latest/api/com/typesafe/config/ConfigFactory.html#defaultOverrides--) (system properties)
  * the app's settings
- * `ConfigFactory.defaultReference()` (reference.conf)
+ * [ConfigFactory.defaultReference()](https://lightbend.github.io/config/latest/api/com/typesafe/config/ConfigFactory.html#defaultReference--) (reference.conf)
 
 The normal goal is to customize the middle layer while leaving the
 other two alone.
 
- * `ConfigFactory.load()` loads the whole stack
- * the overloads of `ConfigFactory.load()` let you specify a
+ * [ConfigFactory.load()](https://lightbend.github.io/config/latest/api/com/typesafe/config/ConfigFactory.html#load--) loads the whole stack
+ * the overloads of [ConfigFactory.load()](https://lightbend.github.io/config/latest/api/com/typesafe/config/ConfigFactory.html#load-java.lang.ClassLoader-com.typesafe.config.ConfigParseOptions-com.typesafe.config.ConfigResolveOptions-) let you specify a
```

Comment:
It's the family of methods. I have added the specifics version. Is it ok?

Reply:
I'm not sure what is the best here. There are many other variants for the `load` method, also for `parse`. Maybe the best here is to only link to the ConfigFactory Javadoc page without pointing to any specific method.